test/common_utils.py

```diff
@@ -28,7 +28,7 @@
 from torchvision.transforms.v2.functional import convert_dtype_image_tensor, to_image_tensor
 
 
-IN_OSS_CI = any(os.getenv(var) == "true" for var in ["CIRCLECI", "GITHUB_ACTIONS"])
+IN_OSS_CI = bool(os.getenv('CI'))
```

Comment:
Not using `"CI"` was a intentional choice: https://github.com/pytorch/vision/pull/6970#discussion_r1029402704

Reply:
> Hey @lcskrishna and thanks for the PR. I'm a little confused about the purpose of it though: you state
> 
> > This PR fixes the ROCm CI to run same tests as CUDA CI.
> 
> but as of now, we aren't running any ROCm CI. What is your use case here as this patch won't do anything for us?

Hi @pmeier,

You are right that there is no ROCm CI on torchvision currently (we hope to change that in the near future!). However, the use-case is us trying to run internal CI on Jenkins for torchvision, and observing that some unit tests are not skipped for ROCm but skipped for upstream because of the above logic specific to CI platforms.

Reply:
> Not using `"CI"` was a intentional choice: [#6970 (comment)](https://github.com/pytorch/vision/pull/6970#discussion_r1029402704)

Honestly, that comment seems to be based on FUD rather than an actual issue with using the "CI" env variable :). We have some real examples of the PyTorch project using the "CI" env var without it interfering with Meta internal systems (afaik):
https://github.com/pytorch/pytorch/blob/master/tools/test/test_upload_test_stats.py#L6
https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_utils.py#L100

I'm not sure if Meta internal systems for torch and torchvision are different, but could we investigate it a bit more before concluding that "CI" won't work? It would make this code more "standard" and CI-vendor-agnostic. Thanks for considering!

Reply:
> You are right that there is no ROCm CI on torchvision currently (we hope to change that in the near future!). However, the use-case is us trying to run internal CI on Jenkins for torchvision, and observing that some unit tests are not skipped for ROCm but skipped for upstream because of the above logic specific to CI platforms.

If this is just temporary and internal issue, can't you simply set one of the environment variables to fake being in such an environment?

> I'm not sure if Meta internal systems for torch and torchvision are different, but could we investigate it a bit more before concluding that "CI" won't work? It would make this code more "standard" and CI-vendor-agnostic. 

I'm in no position to perform such an investigation as I don't have access to Meta internal CI. However, I know that is was very deliberate to separate OSS, i.e. external, and internal CI. This is why we have this flag in the first place.

Reply:
@malfet @fmassa Would you be able to help here? We are trying to figure out if using the `CI` env var instead of vendor-specific env vars would be a feasible solution to keep Meta internal CI systems working?

Reply:
@NicolasHug add this logic. But before we start any investigations, could you please answer my question above?

> If this is just temporary and internal issue, can't you simply set one of the environment variables to fake being in such an environment?

Reply:
@lcskrishna  @jithunnair-amd thanks for the PR.

The pytorch and torchvision CIs are setup somewhave differently both in OSS and internally, so something that works in pytorch core may not work the same way for torchvision.

As @pmeier mentioned, the explicit use of `"CIRCLECI", "GITHUB_ACTIONS"` instead of just `"CI"` is here to prevent any potential conflict with the internal CI. For example the internal CI may at some point decide to *also* set that variable on internal infra; that could lead to silent error or tests being skipped and it could take a while to diagnose, so we want to avoid such a scenario.